<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="sa_area">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>301</width>
         <height>481</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_start_all">
      <property name="text">
       <string>Запустить все</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_restart_all">
      <property name="text">
       <string>Перезапустить все</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_stop_all">
      <property name="text">
       <string>Остановить все</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
